class_generator/tests/manifests/ServingRuntime/serving_runtime.py

```diff
@@ -0,0 +1,161 @@
+# Generated using https://github.com/RedHatQE/openshift-python-wrapper/blob/main/scripts/resource/README.md
+
+from typing import Any, Dict, List, Optional
+from ocp_resources.resource import NamespacedResource, MissingRequiredArgumentError
+
+
+class ServingRuntime(NamespacedResource):
+    """
+    No field description from API; please add description
+    """
+
+    api_group: str = NamespacedResource.ApiGroup.SERVING_KSERVE_IO
+
+    def __init__(
+        self,
+        affinity: Optional[Dict[str, Any]] = None,
+        spec_annotations: Optional[Dict[str, Any]] = None,
+        built_in_adapter: Optional[Dict[str, Any]] = None,
+        containers: Optional[List[Any]] = None,
+        disabled: Optional[bool] = None,
+        grpc_data_endpoint: Optional[str] = "",
+        grpc_endpoint: Optional[str] = "",
+        http_data_endpoint: Optional[str] = "",
+        image_pull_secrets: Optional[List[Any]] = None,
+        spec_labels: Optional[Dict[str, Any]] = None,
+        multi_model: Optional[bool] = None,
+        node_selector: Optional[Dict[str, Any]] = None,
+        protocol_versions: Optional[List[Any]] = None,
+        replicas: Optional[int] = None,
+        storage_helper: Optional[Dict[str, Any]] = None,
+        supported_model_formats: Optional[List[Any]] = None,
+        tolerations: Optional[List[Any]] = None,
+        volumes: Optional[List[Any]] = None,
+        **kwargs: Any,
+    ) -> None:
+        """
+        Args:
+            affinity (Dict[str, Any]): No field description from API; please add description
```

Comment:
Can we update doc string with proper helper description for the use to understand better

Reply:
We can copy the desc from other pr and paste it if that make sense

Reply:
The user should update this since the API does not describe this arg.

Reply:
@myakove I understand that’s why I suggested adding it here. Are you suggesting that user who uses this resource to defined in their code instead of defining in the library? If so, I don’t think that’s a good idea. It’s important to maintain what’s being created in this repo, so I believe modifying it here makes more sense, in my opinion.

Reply:
@tarukumar this is just the tests part; my bad. after this pr is merged, you can generate the resource and add description

Reply:
okay